CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All changes to `doctl` will be documented in this file.
 
+## [1.14] - UNRELEASED
```

Comment:
A set of small changes (it's confusing the first time - it will feel ridiculously simple next time!)

* The release number should be `1.14.0`
* Since 1.13.1 was not released, please fold all the changes into `1.14.0`
* Add the author of #408
* Replace `UNRELEASED` with today's date

Reply:
Looks great!